examples/terraform/vsphere_centos/README.md

```diff
@@ -0,0 +1,106 @@
+# vSphere Quickstart Terraform configs
```

Comment:
TODO: extend README to mention that those configs are for CentOS-based operating systems.